packages/grafana-ui/src/Intro.mdx

```diff
@@ -2,11 +2,11 @@ import { Meta } from '@storybook/blocks';
 
 <Meta title="Docs Overview/Intro" parameters={{ options: { isToolshown: false }, id: 1 }} />
 
-# Grafana design system
+# Grafana React Components
 
 > **@grafana/ui is currently in BETA**.
```

Comment:
Should we remove this part? I don't think it's in beta anymore.

Reply:
If so we should also remove it from the [README](https://github.com/grafana/grafana/blob/main/packages/grafana-ui/README.md).

Reply:
Yeah I think we should remove it from both of these 👍